osh.spec

```diff
@@ -241,13 +248,15 @@ fi
 %{_sbindir}/osh-retention
 %{_sbindir}/osh-stats
 %{_sysconfdir}/osh/hub
-%exclude %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py
-%exclude %{python3_sitelib}/osh/hub/scripts/umb-emit.py
+%exclude %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py*
+%exclude %{python3_sitelib}/osh/hub/scripts/umb-emit.py*
 %{python3_sitelib}/osh/hub
 %{_unitdir}/osh-retention.*
 %{_unitdir}/osh-stats.*
 %exclude %{python3_sitelib}/osh/hub/settings_local.py*
 %exclude %{python3_sitelib}/osh/hub/__pycache__/settings_local.*
+%exclude %{python3_sitelib}/osh/hub/__pycache__/settings.*
```

Comment:
Shall this cache be kept? Where/how it should be moved?

Reply:
This glob is a superset of the above glob, isn't it?  None if them is needed I think.  `settings.py` is installed by `osh-hub` and the other files should not byte-compiled at this location at all.